plugin.video.vimeo/resources/lib/api.py

```diff
@@ -0,0 +1,413 @@
+import json
```

Comment:
Init empty placeholder files shouldn't be needed anymore with python3

Reply:
Thanks for the info, didn't know that. Should I remove them from this PR?

Reply:
If you want yes, not mandatory. I didn't merge it to give you time to do the change. Up to you, you can do it later as well.

Reply:
I'll do it later, thanks! Can be merged.